mesa/examples/basic/conways_game_of_life/app.py

```diff
@@ -1,12 +1,20 @@
+import os.path
+import sys
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../../"))
+)
+
+
```

Comment:
This section was only used for testing, right? Or do we want it in every example? (might help, I also add it often)

Don't mind what as long as it's consistent.

If you want to remove it:
```suggestion
```